build_scripts/shared_versions.sh

```diff
@@ -30,7 +30,6 @@ PG_WANTED_EXTENSIONS=(
 # eventually updated for pg19
 SKIP_FOR_PG18=(
     pg_stat_monitor
-    rum
     ai
     timescaledb
```

Comment:
probably want to remove timescaledb now!

Reply:
also cicd/shared.sh has a couple block for timescaledb.so + pg18, and possibly a couple other blocks (commit f943939cfa341d5af8e7dd64d746921cfecb8695 might help)